internal/providers/terraform/aws/tags.go

```diff
@@ -49,6 +49,14 @@ func ParseTags(defaultTags *map[string]string, r *schema.ResourceData) *map[stri
 			keysAndValues = append(keysAndValues, k, v)
 		}
 
+		if r.Type == "aws_instance" {
+			for k, v := range *defaultTags {
```

Comment:
Are we sure this works correctly? I didn't think terraform applied default tags to volume tags like this.

Reply:
yup after some testing with AWS this doesn't seem to actually be the case

Reply:
No sorry, default tags are actually propagated to the volume_tags but only in `v5.39.0` and above https://github.com/hashicorp/terraform-provider-aws/issues/19890. We'll have to write some logic to detect the provider version and propagate accordingly.